app/templates/zazurc.json

```diff
@@ -1,6 +1,7 @@
 {
   "hotkey": "alt+space",
   "theme": "tinytacoteam/zazu-light-theme",
+  "primaryMonitor": false,
```

Comment:
Instead of `primaryMonitor` what do you think about `position` or `placement` ? With values such as `primary` or `detect` ?

If we want to keep it to a `boolean` we could maybe do `forcePrimaryMonitor` maybe?

cc @afaur @JeroenBoersma @amjith what do you all think about the name here?